### ISSUE TITLE
Add element to form array: Warning: Cannot update a component: (`ReactFinalForm`) while rendering a different component (`Field`)

### ISSUE THREAD

**Issue by dominikzogg** [final-form/final-form-arrays#50]
### Are you submitting a **bug report** or a **feature request**?

bug report

### What is the current behavior?

I get the following warning running jest test, since updating to react 16.13.0:
https://reactjs.org/blog/2020/02/26/react-v16.13.0.html#warnings-for-some-updates-during-render

```
    console.error node_modules/react-dom/cjs/react-dom.development.js:88
      Warning: Cannot update a component (`ReactFinalForm`) while rendering a different component (`Field`). To locate the bad setState() call inside `Field`, follow the stack trace as described in https://fb.me/setstate-in-render
          in Field (at FormField.tsx:19)
          in div (created by FormField)
          in FormField (at FormField.tsx:17)
          in FormField (at PetForm.tsx:38)
          in div (at PetForm.tsx:37)
          in FieldArray (at PetForm.tsx:34)
          in div (created by FormField)
          in FormField (at PetForm.tsx:32)
          in form (created by Form)
          in Form (at PetForm.tsx:30)
          in ReactFinalForm (at PetForm.tsx:17)
          in PetForm (at PetForm.test.tsx:120)
```

https://travis-ci.org/github/chubbyphp/petstore-react
https://github.com/chubbyphp/petstore-react/blob/master/src/__tests__/Component/Form/PetForm.test.tsx#L106-L158

### What is the expected behavior?

Run without issue

### What's your environment?

node: v12.16.1

"react": "^16.13.1",
"react-dom": "^16.13.1",
"react-final-form": "^6.3.5",
"react-final-form-arrays": "^3.1.1"

**Comment by dominikzogg** [comment]
added the issue in the wrong repository

**Comment by erikras** [comment]
Published a fix in [`react-final-form@6.4.0`](https://github.com/final-form/react-final-form/releases/tag/v6.4.0) and [`final-form@4.19.0`](https://github.com/final-form/final-form/releases/tag/v4.19.0). That should fix this issue.

Read more: https://github.com/final-form/react-final-form/issues/751#issuecomment-606212893
